docs/docs-beta/docs/guides/deployment/kubernetes.md

```diff
@@ -1,4 +1,220 @@
 ---
 title: "Deploy to Kubernetes"
 sidebar_position: 21
----
\ No newline at end of file
+---
+
+Deploying Dagster to a Kubernetes cluster is a popular option for maintaining a production Dagster deployment. Dagster uses Helm, a package manager for Kubernetes applications, to help manage deploying Dagster to a Kubernetes cluster.
```

Comment:
> a production Dagster deployment

I think we should position this (and other OSS deployment) docs as reference/examples for how to run the Dagster-specific parts of a productionized Dagster deployment.

Productionizing it for real will probably require additional decisions around networking, access control, running persistent data stores, monitoring, etc. that are all out of scope for the Dagster project and have too large of a decision matrix for us to fully capture.

Reply:
I think I agree.  The goal of this guide shouldnt be to teach you how to build production systems, but rather, what the components of a Dagster production deployment are.

Reply:
ok i can reframe this intro. Do you think there are other systemic changes that need to be made to the doc to position it this way?

Reply:
is a doc that discusses thing like networking, access control data stores, etc something that we plan to do?